### PR TITLE
Add upgrade guide for `@modelClass` and `@model` in v5

### PR DESCRIPTION
**Pull request by lorado**
Just improving upgrade guide.

### CODE REVIEW

**Comment by spawnia** [comment]
Great idea. I always like to add codeblocks tagged with `diff` and show a concrete example of before/after the upgrade, can you add that here as well?

**Comment by lorado** [comment]
I am not sure about examples... Is it ok?

File: UPGRADE.md [hunk 1]
```diff
@@ -34,3 +34,23 @@ Client queries will have to be changed like this:
...
+    id: ID! @globalId
+}
+```
+
+### `@modelClass` is renamed to `@model`
+
+```diff
+-type PaginatedPost @modelClass(class: "\\App\\Post") {
++type PaginatedPost @model(class: "\\App\\Post") {
+    id: ObfId!
```
Comment (spawnia) [inline UPGRADE.md]:
```suggestion
    id: ID!
```

Comment (lorado) [reply UPGRADE.md]: Gosh... sorry, forgot to change ID type

File: UPGRADE.md [hunk 2]
```diff
@@ -34,3 +34,23 @@ Client queries will have to be changed like this:
 
 If you absolutely cannot break your clients, you can re-implement `@orderBy` in your
 project - it is a relatively simple `ArgManipulator` directive.
+
+### `@model` directive was repurposed 
+
+`@model` directive in v5 does what `@modelClass` did in v4. To implement similar functionality of `@model` from v4 use `@node` directive instead.
```
Comment (spawnia) [inline UPGRADE.md]: I think this part is somewhat confusing. How about we just give the minimal information needed to upgrade and link https://github.com/nuwave/lighthouse/pull/974 to give background information.

The upgrade guide should mention concrete steps, those are to 1. rename `@model` to `@node`  and 2. rename `@modelClass` -> `@model` (in that order).

Comment (lorado) [reply UPGRADE.md]: Would a comment like "Please apply changes in your project step by step from up to down" be enough? Alternatively we can define number to steps, and they then have to be applied in the incremental order.

Comment (spawnia) [reply UPGRADE.md]: I tried a bit different approach, what do you think?

Comment (lorado) [reply UPGRADE.md]: Oh yes, that is much better!

**Pull request merged**